helm/inteli-api/templates/configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     {{- include "inteli-api.labels" . | nindent 4 }}
 data:
   port: {{ .Values.config.port | quote }}
+  dscpApiHost: {{ .Values.config.dscpApiHost }}
```

Comment:
are values for this supposed to be formatted as `http://service:port/` ?
If it is just a hostname then please add a section for the port as well

Reply:
it should have port as well, I'll add. `ct install` wasn't complaining since port has a default so I missed it

Reply:
Maybe the port you've used should be a `devDefault` instead of a default?

Reply:
done